### PR TITLE
Feature/localization

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 /*
  * Update dialog titles/messages
  */
-"MSDistributeAppUpdateAvailable" = "Доступно обновление";
-"MSDistributeAppUpdateAvailableMandatoryUpdateMessage" = "Вышло обязательное обновление. Вам необходимо обновиться до %@ %@ (%@).";
-"MSDistributeAppUpdateAvailableOptionalUpdateMessage" = "%@ %@ (%@) доступно для загрузки и установки.";
-"MSDistributeInAppUpdatesAreDisabled" = "Обновления из приложения отключены.";
+"MSDistributeAppUpdateAvailable" = "Доступно обновление приложения";
+"MSDistributeAppUpdateAvailableMandatoryUpdateMessage" = "Разработчики этого приложения требуют выполнить обновление до %@ %@ (%@).";
```
Comment (MatkovIvan) [inline MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings]: This sounds much worse than before

File: MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings [hunk 2]
```diff
@@ -1,15 +1,15 @@
 /*
  * Update dialog buttons
  */
-"MSDistributeUpdateNow" = "Обновить сейчас";
-"MSDistributeAskMeInADay" = "Спросить завтра";
-"MSDistributeViewReleaseNotes" = "Просмотреть примечания к обновлению";
+"MSDistributeUpdateNow" = "Обновить";
+"MSDistributeAskMeInADay" = "Спросить через день";
```
Comment (MatkovIvan) [inline MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings]: Better to leave it as it was

Comment (jaelim-ms) [reply MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings]: This is copied form TouchDown. I know this doesn't look good but it needs to be addressed in a separate PR. @jwallra please update the file in `Localization/`

File: MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings (hunk 1 above)
Comment (jaelim-ms) [reply MobileCenterDistribute/MobileCenterDistribute/Resources/ru.lproj/MobileCenterDistribute.strings]: This is copied form TouchDown. I know this doesn't look good but it needs to be addressed in a separate PR. @jwallra please update the file in `Localization/`

File: Sasquatch/Sasquatch/cs.lproj/Sasquatch.strings [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/*
+  Sasquatch.strings
+  Sasquatch
+*/
+"distribute_alert_title" = "New update available!";
+"distribute_alert_message" = "This is a customized dialog. - Czech";
+"distribute_alert_yes" = "Yes";
+"distribute_alert_no" = "No";
```
Comment (jwallra) [inline Sasquatch/Sasquatch/cs.lproj/Sasquatch.strings]: We're not localizing sasquatch strings, why are these files here?

Comment (jaelim-ms) [reply Sasquatch/Sasquatch/cs.lproj/Sasquatch.strings]: This is required because the app supports all localized languages in Distribute.
